flake.nix

```diff
@@ -0,0 +1,33 @@
+{
+  description = "A command line interface for Pleasant Password Server";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+      in
+      {
+        packages.default = pkgs.buildGoModule {
+          pname = "pleasant-cli";
+          version = "0.8.1";
```

Comment:
Is it possible here to get the version number externally, rather than configuring it directly in this file? The version number currently has its single source of truth inside of `cmd/root.go`.

https://github.com/marevers/pleasant-cli/blob/868b8b66b1f390886713dde40b7d2fa92835a433/cmd/root.go#L31

I see two options:

1) Retrieve the version number from that file.
2) Instead of storing the version number in that file, we add a separate `VERSION` file in the root of the repository which holds the version number. For the Go build process, I'll make a change that reads the version number from there instead of directly from the variable in `cmd/root.go`.